File: 1-TwoSum/1-TwoSum.py
# Last updated: 8/2/2025, 4:05:55 PM
class Solution(object):
    def twoSum(self, nums, target):
        for i in range(len(nums)-1):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    return [i,j]
                    
            
        
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: 1-TwoSum/1-TwoSum.java
// Last updated: 9/22/2025, 9:34:02 PM
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] l=new int[2];
        for(int i=0;i<nums.length;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    l[0]=i;
                    l[1]=j;
                    break;
                }
            }
        }
        return l;
    }
}